MREUnityRuntime/MREUnityRuntimeLib/Core/Interfaces/ICollider.cs

```diff
@@ -28,8 +28,8 @@ public interface ICollider
         //CollisionLayer CollisionLayer { get; }
 
         /// <summary>
-        /// The type of the collider.  <see cref="ColliderType"/>
+        /// The type of the collider.  <see cref="Shape"/>
```

Comment:
>type [](start = 16, length = 4)

Shape

Reply:
Still should say shape in the comment

---
In reply to: [313531621](https://github.com/microsoft/mixed-reality-extension-unity/pull/98#discussion_r313531621) [](ancestors = 313531621)

Reply:
Please update this to say "The shape of the collider." before submitting.